### DISCUSSION TITLE
In the logs, the jsons are not prettyfied

### DISCUSSION THREAD

**Discussion by avifatal** [mockoon/mockoon#367] [Help] [answered]
Hi,
When I'm looking at the logs. I try to view the JSON body.
The colors are working, but it appears to be in one row, which is impossible to read.
Is this a bug? 
Images attached below.
Thanks

![image](https://user-images.githubusercontent.com/12430740/99281518-432aec00-283b-11eb-9888-b6a094f3370a.png)

![image](https://user-images.githubusercontent.com/12430740/99281621-681f5f00-283b-11eb-9107-001e8487c59b.png)

**Comment by 255kb** [comment, marked as answer]
No it's not a bug, in the environment logs page, it's showing the raw body content on purpose, most of the time truncated (truncation limit can be set in the settings). 
But in the modal, where there is an editor, it's true that we could do some formatting. Currently we show what is coming from the request or is being sent in the response, which obviously has no line returns. 
Could you please open an issue? 
It may be solved anyway for the next version as I need to migrate the app to Angular 11 and unfortunately the editor we use is not compatible. So, we may have to switch to something else, maybe Monaco (VS Code's editor), which could offer more options (formatting, etc).

**Reply by avifatal** [reply]
Sure. I'll report that. btw, here you can see that it is prettified, https://mockoon.com/docs/latest/requests-logging/ this is why I thought it is a bug.

**Reply by 255kb** [reply]
True, but in this case the response body was probably sent with the formatting.
